cdm/core/src/test/java/ucar/nc2/time/TestCalendars.java

```diff
@@ -148,6 +148,23 @@ public void testZeroYear() {
     assert unit2.getCalendar() == Calendar.proleptic_gregorian;
   }
 
+  @Test
+  public void testWhitespaceCompatibilityWithUdunits() {
+    String date = "2022-1-1";
+    String time = "12:34:00Z";
+    String[] udunitsWhitespace = new String[] {" ", "  ", "\t", "\n", "\r", "\f"};
+    for (Calendar cal : Calendar.values()) {
+      // unit with time in ISO format to use for testing
+      CalendarDateUnit isoFormatUnit =
+          CalendarDateUnit.withCalendar(cal, String.format("secs since %sT%s", date, time));
+
+      for (String whitespace : udunitsWhitespace) {
+        CalendarDateUnit testUnit =
+            CalendarDateUnit.withCalendar(cal, String.format("secs since %s%s%s", date, whitespace, time));
+        assert (isoFormatUnit.getBaseDate().equals(testUnit.getBaseDate()));
```

Comment:
I know all the other tests here are using the built in `asserts` but we would like to eventually have them all use google truth asserts, something like:
```
        assertThat(testUnit.getBaseDate()).isEqualTo(isoFormatUnit.getBaseDate());
```
(you would probably also need to add `import static com.google.common.truth.Truth.assertThat;`)

Reply:
Got it, thanks! I just updated it and pushed the change